src/containers/profile/Profile.js

```diff
@@ -1,27 +1,55 @@
 import React, { useState, useEffect } from "react";
 import GithubProfileCard from "../../components/githubProfileCard/GithubProfileCard";
-import axios from "axios";
+import ApolloClient from "apollo-boost";
```

Comment:
Instead of two import statement from same library make it as a single one. See the suggested changes.
```suggestion
import ApolloClient, { gql } from "apollo-boost";
```